manifests/apache/vhost.pp

```diff
@@ -125,10 +125,11 @@
   }
 
   if $enable_ldap_auth {
-    $ldap_additional_includes = [ "${puppetboard::params::apache_confd}/puppetboard-ldap.conf" ]
-    $ldap_require = File["${puppetboard::params::apache_confd}/puppetboard-ldap.conf"]
-    file { "${puppetboard::params::apache_confd}/puppetboard-ldap.conf":
+    $ldap_additional_includes = [ "${::puppetboard::params::apache_confd}/puppetboard-ldap.part" ]
```

Comment:
please don't add `::` between `$` and the variable. That is deprecated puppet 3 syntax.

Reply:
Ah yep, fixed.